doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2805,6 +2814,12 @@ Matter samples
 
 The issues in this section are related to :ref:`matter_samples`.
 
+.. rst-class:: v2-9-0
+
+KRKNWK-19861: ICD DSLS does not work with the Smoke CO Alarm
+  According to the sample's readme the ICD DSLS can be enabled by setting :kconfig:option:`CONFIG_CHIP_ICD_DSLS_SUPPORT` Kconfig option to ``y``.
+  There is a missing information that additionally the ICD Management cluster's feature map has to be set to ``0xf`` in the sample's ``.zap`` file.
```

Comment:
```suggestion
  Additionally, the ICD Management cluster's feature map has to be set to ``0xf`` in the sample's ``.zap`` file.
  This piece of information is missing from the documentation.
  It will be added in the next version.
```